libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -184,6 +184,21 @@
         LOGIN_INSERTED_INVALID_URL,
         LOGIN_AUTOFILL_CREDENTIALS_FILLED,
         LOGIN_AUTOFILL_CREDENTIALS_UPDATED,
+        LOGIN_PROLOGUE_VIEWED,
+        LOGIN_EMAIL_FORM_VIEWED,
+        LOGIN_EMAIL_RETRY_VIEWED,
+        LOGIN_MAGIC_LINK_OPEN_FORM_VIEWED,
+        LOGIN_MAGIC_LINK_REQUEST_FORM_VIEWED,
+        LOGIN_PASSWORD_FORM_VIEWED,
+        LOGIN_PASSWORD_RETRY_VIEWED,
+        LOGIN_URL_FORM_VIEWED,
+        LOGIN_URL_RETRY_VIEWED,
```

Comment:
When I first came up with the list of events and keys I was basing them off of what we had in the mural board.  I understood the mural board to show a separate screen for error messages, which is what the "RETRY" events represented.  The final visual design makes it clear that errors are shown on the same screen vs a new screen so we can get rid of these three keys.  Sorry for the fuss here.